integration/internal/swarm/service.go

```diff
@@ -20,6 +23,67 @@ const (
 	defaultSwarmPort = 2477
 )
 
+// ServicePoll will tweak the pollSettings for `service`
+func ServicePoll(config *poll.Settings) {
+	// Override the default pollSettings for `service` resource here ...
+
+	if runtime.GOARCH == "arm64" || runtime.GOARCH == "arm" {
+		config.Timeout = 1 * time.Minute
```

Comment:
I guess we don’t have tests that already set a longer timeout? Otherwise we may have to check first if the existing timeout is longer

Reply:
The default timeout value is 10s configured in the 3rd party code `github.com/gotestyourself/gotestyourself/poll/poll.go`, for each `poll.WaitOn()` calling, we can override the timeout value with the default function defined here. On Arm platforms, for those multiple-service creation, the 1min value is reasonable and the CI can pass all the test cases. Let me know if any suggestions 😄